src/components/DayPicker.jsx

```diff
@@ -301,10 +305,12 @@ export default class DayPicker extends React.Component {
         didTransitionMonth = this.maybeTransitionPrevMonth(newFocusedDate);
         break;
       case 'Home':
+        if (e) e.preventDefault();
```

Comment:
Awesome, let's do that - we can pass `{ preventDefault() {} }` in tests if needed.